sdk/go/request.go

```diff
@@ -0,0 +1,81 @@
+package feast
+
+import (
+	"fmt"
+	"github.com/gojek/feast/sdk/go/protos/feast/serving"
+	"github.com/golang/protobuf/ptypes/duration"
+	"github.com/golang/protobuf/ptypes/timestamp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var (
+	ErrInvalidFeatureName       = "Invalid feature name %s provided, feature names must be in the format featureSet:version:featureName."
+)
+
+// OnlineFeaturesRequest wrapper on feast.serving.GetOnlineFeaturesRequest.
+type OnlineFeaturesRequest struct {
+
+	// Features is the list of features to obtain from Feast. Each feature must be given by its fully qualified ID,
+	// in the format featureSet:version:featureName.
+	Features      []string
+
+	// MaxAgeSeconds is the maximum allowed staleness of the features in seconds. This max age will be applied to all
+	// featureSets in this request.
+	// Setting this value to 0 will cause feast to default to the max age specified on the feature set spec, if any.
+	MaxAgeSeconds int
```

Comment:
Should we leave this out for now? Users can request features from multiple feature sets, and they will definitely have different maximum ages. So IMHO either we need a way to set this at the feature set level, or we should just leave it to default (registered max age).

Reply:
Ok. We need to KIV though